jenkins/Deploy.groovy.j2

```diff
@@ -47,7 +47,7 @@ def deploy_docs() {
       git status
       git checkout -B $DOCS_DEPLOY_BRANCH
 
-      rm -rf docs
+      git ls-tree HEAD docs/ --name-only | grep -vP '^docs/v\\d' | xargs rm -rf
```

Comment:
is \\ right?

Reply:
it goes through Jenkins' parser first so the `\\` has to be escaped, bash sees `v\d` as it should

Reply:
ah gotcha